www/getj00/cv06/index.php

```diff
@@ -0,0 +1,97 @@
+
+<?php
+const TD = '</td><td>';
+const MAXPRICE = 10000;
+
+// is local to the server this is running on
+$pdo=new PDO('mysql:host=localhost;dbname=test;charset=utf8mb4', 'getj00', 'xxx');
+$pdo->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
+$pdo->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+$sql1 = "SELECT * from products WHERE price <= :maxPrice";
+$stmt1 = $pdo->prepare($sql1);
+$stmt1->execute(['maxPrice' => isset($_GET['budget']) ? $_GET['budget'] : MAXPRICE]);
+
+$sql2 = "SELECT * from products_eshop WHERE price <= :maxPrice";
+$stmt2 = $pdo->prepare($sql2);
+$stmt2->execute(['maxPrice' => isset($_GET['budget']) ? $_GET['budget'] : MAXPRICE]);
+
+$sql3 = "SELECT name FROM categories_eshop WHERE category_id = :catid";
```

Comment:
kdyz tech statementu bude treba 50 tak budete mit `$stmt50 = ...`? proto se vyplati to strukturovat do trid.